examples/userdemo/pgf_fonts.py

```diff
@@ -8,18 +8,23 @@
 import matplotlib.pyplot as plt
 plt.rcParams.update({
     "font.family": "serif",
-    "font.serif": [],                    # use latex default serif font
-    "font.sans-serif": ["DejaVu Sans"],  # use a specific sans-serif font
+    # Use LaTeX default serif font.
+    "font.serif": [],
```

Comment:
Note, this fallback does not work for me; it decides to use 'DejaVu Sans' before even getting to pgf/LaTeX. I'm not sure how to fix it.